src/system/router.py

```
@@ -0,0 +1,39 @@
+from fastapi import APIRouter
+import os
+
+
+sys_router = APIRouter(tags=["System"])
+
+
+@sys_router.get("/sys/health")
+async def health():
```

Ruff (0.8.2)

Missing return type annotation for public function `health`

(ANN201)